azure_functions_worker/mmap_handler/file_accessor_windows.py

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import mmap
+from typing import Optional
+from .file_accessor import FileAccessor
+
+
+class FileAccessorWindows(FileAccessor):
```

Comment:
Didn't we have the fanciness where we subclassed this and sent the seeked memory mapped directly after the header?

Reply:
As discussed earlier, we don't need that. It was when we were playing with PyArrow types (but even for that we found a way around it).